namer/namer.cc

```diff
@@ -389,6 +389,11 @@ class SymbolFinder {
                 }
                 if (!original.hasPosArgs()) {
                     ENFORCE(!methodVisiStack.empty());
+
+                    if (original.recv.tag() != ast::Tag::Local) {
```

Comment:
Can you change this to use isSelfReference?

https://github.com/search?q=repo%3Asorbet%2Fsorbet%20isselfreference&type=code

and can you add a test like


```ruby
x = nil
x.private
# ...
```

because the change as written doesn’t work for this case.

Reply:
I updated to use `isSelfReference()` and added a test. Thank you for your really quick response.